### PR TITLE
Split getters package

### PR DESCRIPTION
**Pull request by turbocrime**
for consumption 🤒

### CODE REVIEW

File: apps/webapp/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "zustand": "^4.5.1"
   },
   "devDependencies": {
+    "@penumbra-zone/getters": "workspace:*",
```
Comment (grod220) [inline apps/webapp/package.json:35]: This should be a normal dependency given it's a runtime package and not really dev tooling

File: packages/getters/package.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "version": "1.0.0",
+  "description": "Convenience getters for the deeply nested optionals of Penumbra's protobuf types",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint \"**/*.ts*\"",
+    "test": "vitest run"
+  },
+  "devDependencies": {
+    "bech32": "^2.0.0"
+  }
```
Comment (grod220) [inline packages/getters/package.json:12]: Think this should be a normal dep too

File: packages/getters/asset.ts
Comment (turbocrime) [inline packages/getters/asset.ts:1]: notably, this file does not contain a gitter but would have caused a circular dependency. so i just moved it to getters

**Review by grod220 — APPROVED** [review]
Think a passing build + moving the package to the normal dep section and it's good to merge

**Review by jessepinho — APPROVED** [review]
Nice! 

I'd personally put the getter files (except `index.ts`) underneath a `/src` directory in that package. But up to you.

**Comment by grod220** [comment]
> I'd personally put the getter files (except index.ts) underneath a /src directory in that package. But up to you.

Ah yes, we should do that

**Pull request merged**